std/monitoring/tracing/wire.go

```diff
@@ -222,7 +222,17 @@ func Prepare(ctx context.Context, deps ExtensionDeps) error {
 	)
 
 	deps.Middleware.Add(func(h http.Handler) http.Handler {
-		return otelhttp.NewHandler(h, "",
+		return otelhttp.NewHandler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			// add referer
+			span := t.SpanFromContext(r.Context())
+			if span.IsRecording() {
```

Comment:
Nit: `span.IsRecording()` should not be needed. Because if span is not present, then `SetAttributes` is a safe no-op.